<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="chatBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>430</width>
      <height>371</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string>Messages will appear here...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="messageInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>430</width>
      <height>48</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your message...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>430</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameInput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>430</width>
      <height>28</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Enter your name...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
